labs/Lab2-Security.md

```diff
@@ -96,47 +96,48 @@ belong to its tenant.
     of them.) The domain name is different - it needs to be globally unique. If you
     try a name that is already in use by another tenant, the Azure Management site
     will tell you. Finally, you need to specify the main country or region that will
-    be using this tenant. (Do **not** check the B2C checkbox.)
+    be using this tenant. (Do **not** check the B2C checkbox.) 
+    Click the **tick** at the bottom right when you have filled this in.
 
     ![Add directory](media/AddDirectory.png)
 
-    **Note:** you will need the domain name later. Take a note of the value you
-    entered. (You may find it convenient to open a text editor to collect this, and
-    various other bits of information you'll be needing later.)
-
-    Click the tick at the bottom right when you have filled this in.
+     **Note:** you will need the domain name later. Take a note of the value you
+     entered. (You may find it convenient to open a text editor to collect this, and>
+     various other bits of information you'll be needing later.)
+    
 
-6.  Once the directory is ready, it will appear in your list of tenants:
+6.  Once the directory is ready, it will appear in your list of tenants. Click on it in the name column:
 
     ![New tenant](media/TenantInList.png)
 
-    Click on it in the name column.
+    
 
-7.  The Azure Management site will show the quickstart page for the tenant. We don't
-    want this. Click on the **USERS** label near the top
+7.  The Azure Management site will show the quickstart page for the tenant. You don't
+    want this. Click on the **USERS** label near the topYH
 
     ![Users](media/TenantUsersLabel.png)
 
     This will show a list of all the users in the tenant - currently just you. If you
     want to grant other users access to the management site, you'd do it by going to
     this page and then clicking the **ADD USER** button at the bottom of the page.
-    However, we don't need to do that right now, although feel free to if you want.
+    However, you don't need to do that right now, although feel free to if you want.
 
 
-**Note:** There are now two ways to proceed. In either case, you will create an
+> **Note:** There are now **two** ways to proceed. In either case, you will create an
 Application in your Azure AD tenant, and configure your Azure Kit management web
-site to use it, but you can either get Visual Studio to do all the work for you,
+site to use it. You can either get Visual Studio to do the work for you,
 or you can do it by hand. It's easiest to let Visual Studio do it, and if that's
-what you want, proceed to Part 2. However, that will hide some important details, and
-if you'd like to see exactly what's going on, skip Part 2, and go straight to Part 3.
+what you want, proceed to **Part 2**. However, that will hide some important details, and
+if you'd like to see exactly what's going on, skip *Part 2*, and go straight to **Part 3**.
```

Comment:
Be consistent. Here you've use italics for Part 2, and while I understand the thinking here is that you want to give most emphasis to the part they should do next, you need to be consistent: later on you use bold for a part they should skip.